### ISSUE TITLE
Improve SDMX-JSON Integration of ABS.stats data in Terria Maps

### ISSUE THREAD

**Issue by AnaBelgun** [TerriaJS/terriajs#3199]
- work out a good process for how we can get a robust process to dynamically generate catalogues from stat.abs 
- at the moment we do it all statically

**Comment by AnaBelgun** [comment]
to discuss details with ABS on 29 Jan

**Comment by AnaBelgun** [comment]
TBC who will be assigned after meeting with ABS

**Comment by AnaBelgun** [comment]
Discussed and there are 3 distinct processes: 
1)	Configuring region mapping = adding the regions to the terria region mapping server (to allow any csv-geo-au formatted dataset, mapped against CED 2018 to be visualised on NatMap)
2)	Providing access to boundary-coded datasets = connecting to the CED 2018 data (not geometry) via the SDMX JSON API
3)	Providing access to boundary geometry = showing the regions on the map and allowing people to download the corresponding shapefile)

related issue here: Discussed and new issue here: https://github.com/TerriaJS/terriajs/issues/3203

**Comment by meh9** [comment]
I believe there is a new improved version of SDMX on the way, with ABS upgrading in the "middle of the year". We should connect with ABS again and find out,  and possibly start  writing code to support it.

**Comment by AnaBelgun** [comment]
Ana to follow up with ABS
I did, reply pending

**Comment by AnaBelgun** [comment]
Current implementation: https://github.com/TerriaJS/NationalMap-Catalog/blob/master/doc/abs-sdmx-json.md

**Comment by AnaBelgun** [comment]
@rowanwins to update doco with v2.1 implementation once work on this is complete: https://github.com/PacificCommunity/Terria-PacificMap/issues/2

**Comment by AnaBelgun** [comment]
will close this issue for now, until more progress is made on the PacificMap implementation
